index.js

```diff
@@ -1,34 +1,32 @@
-exports.get = function(belowFn) {
-  var oldLimit = Error.stackTraceLimit;
+export function get(belowFn) {
+  const oldLimit = Error.stackTraceLimit;
   Error.stackTraceLimit = Infinity;
 
-  var dummyObject = {};
+  const dummyObject = {};
 
-  var v8Handler = Error.prepareStackTrace;
+  const v8Handler = Error.prepareStackTrace;
   Error.prepareStackTrace = function(dummyObject, v8StackTrace) {
     return v8StackTrace;
   };
-  Error.captureStackTrace(dummyObject, belowFn || exports.get);
+  Error.captureStackTrace(dummyObject, belowFn || get);
 
-  var v8StackTrace = dummyObject.stack;
+  const v8StackTrace = dummyObject.stack;
   Error.prepareStackTrace = v8Handler;
   Error.stackTraceLimit = oldLimit;
 
   return v8StackTrace;
-};
+}
 
-exports.parse = function(err) {
+export function parse(err) {
   if (!err.stack) {
     return [];
   }
 
-  var self = this;
-  var lines = err.stack.split('\n').slice(1);
-
+  const lines = err.stack.split('\n').slice(1);
   return lines
     .map(function(line) {
       if (line.match(/^\s*[-]{4,}$/)) {
-        return self._createParsedCallSite({
```

Comment:
This is the whole reason I migrated to esm. When this script is used in esm (you can import cjs) this self reference was globalThis and caused runtime issues.